v5_user_service.go

```diff
@@ -0,0 +1,67 @@
+package bybit
+
+import (
+	"net/url"
+	"time"
+)
+
+// V5UserServiceI :
+type V5UserServiceI interface {
+	GetAPIKey() (*V5APIKeyResponse, error)
+}
+
+// V5UserService :
+type V5UserService struct {
+	client *Client
+}
+
+// V5APIKeyResponse :
+type V5APIKeyResponse struct {
+	CommonV5Response `json:",inline"`
+	Result           V5ApiKeyResult `json:"result"`
+}
+
+// V5ApiKeyResult :
+type V5ApiKeyResult struct {
+	ID          string `json:"id"`
+	Note        string `json:"note"`
+	APIKey      string `json:"apiKey"`
+	ReadOnly    int    `json:"readOnly"`
```

Comment:
nits:

I want to express it explicitly with a type.

```suggestion
	ReadOnly    ReadOnly    `json:"readOnly"`
```

ref: https://github.com/hirokisan/bybit/blob/804dbfe4c6a217637abcc976ee7cbc3869e09cca/v5_enum.go#L54-L62